qml/cmakesample_generic.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.0
+
+Rectangle {
+    width: 100
+    height: 100
+    color: "baf"
```

Comment:
If the app gets into loading qml, this will break it.

Reply:
Done in #2